lib/utils/environment.go

```diff
@@ -15,9 +15,15 @@ import (
 // Lines that start with "#" or empty lines are ignored. Assignments are in the
 // form name=value and no variable expansion occurs.
 func ReadEnvironmentFile(filename string) ([]string, error) {
+	// open the users environment file. if we don't find a file, move on as
+	// having this file for the user is optional.
 	file, err := os.Open(filename)
 	if err != nil {
-		return nil, trace.ConvertSystemError(err)
+		err = trace.ConvertSystemError(err)
```

Comment:
This error, no matter what kind, should be printed into stderror but the login process should not terminate just because environment file is (for whatever reason) not accessible.